### PR TITLE
chore: updated the Android Gradle plugin to 3.3.0 (#933)

### LINKED ISSUE #933: Updating the Gradle Plugin
**Describe the chore**
Change the Android Gradle plugin to version 3.3.0 

**Would you like to work on the issue?**

 Yes

### PR DESCRIPTION
**Pull request by RebelsterPi**
Fixes #933

Changes: Updated the android gradle plugin to 3.3.0

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <dimen name="layout_margin_large">16dp</dimen>
     <dimen name="layout_margin_extra_large">32dp</dimen>
     <dimen name="layout_margin_extreme_large">56dp</dimen>
-    <dimen name="layout_margin_search_view">100dp</dimen>
+    <dimen name="layout_margin_search_view">40dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Unnecessary change

Comment (nikit19) [inline app/src/main/res/values/dimens.xml]: @RebelsterPi Please revert this change rest looks good
